docs/UserGuide.adoc

```diff
@@ -101,28 +101,31 @@ Expected output:
 [[UpdateTag]]
 === Updating a transaction’s details: `update`
 
-Format: `update i/INDEX | PREFIX_LETTER/UPDATED_DETAILS`
+Format: `update i/INDEX PREFIX_LETTER/UPDATED_DETAILS ...`
 
 ****
 * Updates the transaction's details at the specified `INDEX`.
 ** `INDEX` refers to the index number currently shown in the UI, and must be a *positive integer*.
 * `PREFIX_LETTER` specifies type of detail to be updated, e.g. `n/` for name.
 * `UPDATED_DETAILS` are the new details of type corresponding to `PREFIX_LETTER` that the transaction is to be updated with.
+* Multiple detail types can be updated at once.
 ****
 
 Examples:
 
-*   `update i/1 | n/Curry Laksa`
-*   `update i/1 | c/11`
+*   `update i/1 n/Curry Laksa`
+*   `update i/1 n/Asam Laksa c/11 t/Dinner t/Food`
 
 Expected output: +
 
 NOTE: Assume 2nd command occurs after 1st
 
-*   `Entry name updated: 1. [-] Curry Laksa ($10.50) [Lunch][Food] +
-Original name was: Laksa`
-*   `Entry cost updated: 1. [-] Curry Laksa ($11) [Lunch][Food] +
-Original cost was: $10.50`
+*   `Updated Transaction: [-] Curry Laksa ($10.50) Date: 09/10/2019 Tags: [Food][Lunch]` +
```

Comment:
We could omit showing the original transaction details. If there are mistakes the user could either `undo` or just run `update` again?